ooniprobe/Utility/TestUtility.h

```diff
@@ -14,6 +14,7 @@
 + (UIColor*)getColorForTest:(NSString*)testName;
 + (UIColor*)getColorForTest:(NSString*)testName alpha:(CGFloat)alpha;
 + (void)downloadUrls:(void (^)(NSArray*))successcb onError:(void (^)(NSError*))errorcb;
++ (void)downloadJson:(NSString*)urlStr onSuccess:(void (^)(NSDictionary*))successcb onError:(void (^)(NSError*))errorcb;
```

Comment:
Could you rewrap this long line? I am having troubles to read it on GitHub

Reply:
I do it but then I don't like how it is shown in XCode, we need to really find a compromise as I think XCode is more important than github

Reply:
How about this: it doesn't matter if you wrap prototypes. The compiler already checks that and it's fine to keep them long (for some smaller definition of fine). We wrap however the implementation (i.e. `.m` files) because that's what needs more careful review?

Reply:
Ok